### PR TITLE
Improve content picker selecting multiple answers for calculated summary

### PR DESCRIPTION
**Pull request by SamGodwin2**
### What is the context of this PR?
Implemented new content picker for calculated summaries.

### How to review 
Looks good, works good, tests good.

### CODE REVIEW

File: eq-author-api/schema/tests/calculatedSummary.test.js [hunk 1]
```diff
@@ -302,7 +304,7 @@ describe("calculated Summary", () => {
     );
   });
 
-  it("should error if an answer is added that is a different type to the one that is already selected", async () => {
+  it("should error if an answers of of inconsistent type", async () => {
```
Comment (tgandrews) [inline eq-author-api/schema/tests/calculatedSummary.test.js]: `it should error if there are more than one answer type selected`?

**Comment by hamishtaplin** [comment]
Need to handle there not being any items to pick from, currently it just opens an empty picker:

Screenshot 2019-07-30 at 16 08 48

Change the copy and disable to the button:

Screenshot 2019-07-30 at 16 12 50

File: eq-author/src/App/page/Design/CalculatedSummaryPageEditor/CalSumContentPicker/ContentSelector.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+`;
+
+const SectionTitle = styled.div`
+  background: #f4f5f6;
+  font-size: 11px;
+  padding: 0.3rem 1rem;
+  text-transform: uppercase;
+  letter-spacing: 0.05em;
+  font-weight: bold;
+  color: #666;
```
Comment (hamishtaplin) [inline eq-author/src/App/page/Design/CalculatedSummaryPageEditor/CalSumContentPicker/ContentSelector.js]: change to `${colors.text}`

File: eq-author/src/App/page/Design/CalculatedSummaryPageEditor/CalSumContentPicker/index.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+`;
+
+const Container = styled.div`
+  background: white;
+`;
+
+const ModalTitle = styled.div`
+  font-weight: bold;
+  font-size: 1.2em;
+  color: ${colors.textLight};
```
Comment (hamishtaplin) [inline eq-author/src/App/page/Design/CalculatedSummaryPageEditor/CalSumContentPicker/index.js]:
```suggestion
  color: ${colors.text};
```

File: eq-author/src/App/page/Design/CalculatedSummaryPageEditor/CalSumContentPicker/index.js [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+  margin-bottom: 0.2em;
+`;
+
+const ModalSubtitle = styled.div`
+  font-size: 1em;
+  color: ${colors.textLight};
```
Comment (hamishtaplin) [inline eq-author/src/App/page/Design/CalculatedSummaryPageEditor/CalSumContentPicker/index.js]:
```suggestion
  color: ${colors.text};
```

File: eq-author/src/App/page/Design/CalculatedSummaryPageEditor/CalSumContentPicker/index.js [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+`;
+
+const Type = styled.span`
+  font-size: 10px;
+  background: #e4e8eb;
+  padding: 0.3em 0.7em;
+  border-radius: 1em;
+  text-transform: uppercase;
+  letter-spacing: 0.05em;
+  color: #333;
```
Comment (hamishtaplin) [inline eq-author/src/App/page/Design/CalculatedSummaryPageEditor/CalSumContentPicker/index.js]:
```suggestion
  color: ${colors.text};
```

**Comment by heofs** [comment]
I get this error when creating a new calculated summary: ```Warning: Failed prop type: The prop `questionnaire` is marked as required in `MovePageModal`, but its value is `undefined`.```.

Also, if you open up validation on a number, for example, and click on `Previous answer` or `Custom` then it does some weird switch back to the old selection before it switches to the selected one again.

**Pull request merged**